src/app/home/home.component.ts

```diff
@@ -68,5 +65,11 @@ export class HomeComponent implements OnInit {
       this.getWeather(this.weatherForm.get('country').value, 5); 
   }
 
+  go() {
+    this.getWeather(
+      this.weatherForm.get('country').value
```

Comment:
I just figured out we can search for cities, so I would replace `country` for `city` or another more representative word

Reply:
You are right! I changed it to 'location'. Sounds good to you? Thanks Majo!